stringResources/en-US/resources.resjson

```diff
@@ -20,5 +20,12 @@
 	"Visual_SankeySettings": "Sankey settigns",
 	"Visual_NodePositions": "Node positions",
 	"Visual_ViewportSize": "Viewport sizes",
-	"Visual_Display_Units": "Display units"
+	"Visual_Display_Units": "Display units",
+	"Visual_Cycles": "Cycles displaying",
+	"Visual_DublicateNodes": "Dublicate nodes",
+	"Visual_Dublicate": "Dublicate",
```

Comment:
Spelling mistake. It must be `Duplicate`.

Please check the whole code and fix this mistake everywhere.